src/Controller/StatusController.php

```diff
@@ -61,8 +63,23 @@ public function statusAction(Request $request)
     private function verifyNoDatabaseCacheIssues(): bool
     {
         try {
-            $pid = new Pid('b006m86d'); //Eastenders
-            $this->get('pps.programmes_service')->findByPidFull($pid);
+            // Eastenders clip
+            $clipPid = new Pid('p04r0jcv');
```

Comment:
I assume clips are never removed?

Reply:
I'm assuming this too based on Ben and Bjorn feedback. Works case scenario we won't be able to see if there is an error in the data.